src/resources/views/crud/fields/upload_multiple.blade.php

```diff
@@ -35,7 +35,7 @@
     @endif
     @endif
 	{{-- Show the file picker on CREATE form. --}}
-	<input name="{{ $field['name'] }}[]" type="hidden" value="">
```

Comment:
Hmmm is this change in `upload_multiple` really related to this PR? Or a mistake?

Reply:
😞